packages/main/src/TabContainer.ts

```diff
@@ -1274,28 +1274,24 @@ class TabContainer extends UI5Element {
 		}
 	}
 
-	async _togglePopover(opener: HTMLElement, setInitialFocus = false) {
+	async _togglePopover(opener: HTMLElement) {
 		this.responsivePopover = await this._respPopover();
 
-		if (this.responsivePopover.isOpen()) {
+		if (this.responsivePopover.open) {
 			this._closePopover();
 		} else {
-			await this._showPopoverAt(opener, setInitialFocus);
+			await this._showPopoverAt(opener);
 		}
 	}
 
-	async _showPopoverAt(opener: HTMLElement, setInitialFocus = false, preventInitialFocus = false) {
-		this._hasScheduledPopoverOpen = true;
+	async _showPopoverAt(opener: HTMLElement) {
 		this._setPopoverItems(this._getPopoverItemsFor(this._getPopoverOwner(opener)));
 		this.responsivePopover = await this._respPopover();
 
-		if (setInitialFocus) {
-			this._setPopoverInitialFocus();
-		}
+		this._setPopoverInitialFocus();
 
-		if (this._hasScheduledPopoverOpen) {
```

Comment:
`_hasScheduledPopoverOpen` is needed to avoid popover opening and closing like this:
![2024-05-09_12-09-50](https://github.com/SAP/ui5-webcomponents/assets/32839090/6fe8a755-1645-4d67-b7c9-52336b37ba50)